src/h5web/vis-packs/core/shared/Html.tsx

```diff
@@ -70,20 +85,32 @@ const Html = forwardRef<HTMLDivElement, HtmlProps>((props, ref) => {
       if (parentElement) {
         el.remove();
       }
+
       ReactDOM.unmountComponentAtNode(el);
     };
-  }, [el, parentElement]); // eslint-disable-line react-hooks/exhaustive-deps
+  }, [el, parentElement]);
 
+  // Update size and overflow of container `div`
   useLayoutEffect(() => {
+    el.style.width = `${width}px`;
+    el.style.height = `${height}px`;
+    el.style.overflow = followCamera ? 'hidden' : 'visible';
```

Comment:
Perhaps it should be a dedicated boolean ? Like `limitToCanvas` ?

Reply:
Yeah, I did think about that, but in practice, would we ever want the overflow to be visible when following the camera, or hidden when not following the camera? I can't think of a use case where the two settings need to be decoupled, so I'm thinking that we should wait until one comes along.

Reply:
Sensible :+1: